### PR TITLE
working on listing view and update snap

### PR DESCRIPTION
**Pull request by wkbkhard**
Worked on #1188 to make listing view more useable

### CODE REVIEW

File: src/components/theme/View/ListingView.jsx [hunk 1]
```diff
@@ -20,22 +20,23 @@ const ListingView = ({ content }) => (
     <Helmet title={content.title} />
     <section id="content-core">
       {content.items.map(item => (
-        <article key={item.url}>
-          <h2>
-            <Link to={item.url} title={item['@type']}>
-              {item.title}
-            </Link>
-          </h2>
+        <Segment className="listing-item">
```
Comment (sneridagh) [inline src/components/theme/View/ListingView.jsx]: @wkbkhard the key is missing.

**Comment by sneridagh** [comment]
@wkbkhard upps something went wrong with syncing with master and the changed files are messed up? It sees 220 files changed!

**Comment by wkbkhard** [comment]
I guess it was the rebase :/ 

I'll try to make a new branch and a new pull request on monday, when I am back from my holidays!

**Comment by wkbkhard** [comment]
Now in Pullrequest https://github.com/plone/volto/pull/1235

**Pull request closed without merging**